HISTORY.rst

```diff
@@ -8,6 +8,9 @@ Pending Release
 
 .. Insert new release notes below this line
 
+* Support cases where the function or class name is not in the freevars. This
+  allows patching `__init__` of a class, or patching a recursive module-level function.
```

Comment:
double backticks for code objects in rst

Reply:
done